web/elm/src/Dashboard/Dashboard.elm

```diff
@@ -848,15 +855,7 @@ welcomeCard session { pipelines } =
                 []
 
         noPipelines =
-            case pipelines of
-                None ->
-                    False
-
-                Cached p ->
-                    List.isEmpty p
-
-                Fetched p ->
-                    List.isEmpty p
+            pipelines |> Maybe.map List.isEmpty |> Maybe.withDefault False
```

Comment:
haha nice catch. it doesn't matter what you put there because that function actually never gets called when `pipelines` is `Nothing` -- it's dead code! ccdfe5f167b37f37d0bd5ceb0f964a17856de6da